src/dropdown/_dropdown.scss

```
@@ -43,7 +43,7 @@ $dropdown-header-horizontal-padding: 8px;
   width: 100%;
   height: var(--dropdown-header-button-height);
 
-  background-color: white;
+  background-color: var(--dropdown-header-button-bg);
   border: var(--dropdown-header-button-border-width) solid var(--default-border-color);
```

Should we have a variable for border-style as well ? 😄

---

Yeah maybe we can do something like this

```css
  --dropdown-header-button-border: var(--dropdown-header-button-border-width) solid var(--default-border-color)
```

```suggestion
  border: var(--dropdown-header-button-border);
```